src/lib/example.js

```diff
@@ -135,15 +135,32 @@ async function setProjectName(projDir) {
     'PROJECT_NAME',
     titleCase(name)
   );
+
   replaceInFile(
     path.join(projDir, 'package.json'),
     'package-name',
     kebabCase(name)
   );
 
+  addStartScript(path.join(projDir, 'package.json'));
+
   spin.succeed(_green(step));
 }
 
+/**
+ * Helper to add start script to package.json.
+ * @param {string} file    Path to file
+ * @param {string} script  Start script to add.
+ */
+function addStartScript(file) {
+  const startScript = 'node build/src/index.js';
+  let packageJsonContent = JSON.parse(fs.readFileSync(file, 'utf8'));
```

Comment:
We're using [fs-extra](https://www.npmjs.com/package/fs-extra), so we can simplify the value to `fs.readJsonSync(file)` ([src](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/readJson-sync.md))